check_manifest.py

```diff
@@ -377,11 +378,24 @@ def get_vcs_files():
 
 
 def normalize_names(names):
+    """Normalize file names.
+    """
+    return list(map(normalize_name, names))
+
+
+def normalize_name(name):
     """Some VCS print directory names with trailing slashes.  Strip them.
 
     Easiest is to normalize the path.
+
+    And encodings may trip us up too, especially when comparing lists
+    of files.  Plus maybe lowercase versus uppercase.
     """
-    return [os.path.normpath(name) for name in names]
+    name = os.path.normpath(name)
+    name = os.path.normcase(name)
+    name = unicodify(name)
+    name = unicodedata.normalize('NFC', name)
```

Comment:
Thanks for the links, interesting stuff.

I have modified the function to only do the `unicodedata.normalize` when the platform is darwin.

That should make you more comfortable including this pull request. If you are still not comfortable, that's okay, I can live with this minor issue. :-) (Another approach for this particular project if I want to get rid of this warning by check-manifest would be to remove the non-ascii test file and just create or rename it on the fly when running the tests.)

I tried normalizing to NFD instead of NFC, but then those tests simply fail with basically the error that I am trying to fix here. So seems NFC is the way to go. At least that is true for the tests and that one file when using my Mac.

I could create a separate normalize_name function for Mac, and override the standard normalize_name function when platform is darwin. That would avoid checking the platform once for each time the function is called.  But I timed it with and without that condition on a project with about a thousand files and there it did not matter.